aerich/utils.py

```diff
@@ -101,3 +103,34 @@ def import_py_file(file: Union[str, Path]) -> ModuleType:
     module = importlib.util.module_from_spec(spec)  # type:ignore[arg-type]
     spec.loader.exec_module(module)  # type:ignore[union-attr]
     return module
+
+
+def reorder_m2m_fields(old_m2m_fields: list[dict], new_m2m_fields: list[dict]) -> None:
+    """
+    Reorder m2m fields to help dictdiffer.diff generate more precise changes
```

Comment:
A few comments:
- That's a lot of code to sort table names. I couldn't understand what it is doing after starring at it for a while. It would definitely help to have a comment or comments that explains what is happening.
- That's generally a bad practice to modify the arguments of a function. It would be better to return sorted results.
- Can we sorting on the tortoise side to reduce the amount of code here?

Reply:
> * Can we sorting on the tortoise side to reduce the amount of code here?

This PR is to fix bug, optimize job like reduce amount of code can be later or next version.

Reply:
@henadzit Cloud you review it again?